partner_communication_switzerland/models/contracts.py

```diff
@@ -266,8 +266,31 @@ def _send_reminders_for_birthday_in_1day_or_2months(self):
         tomorrow = today + relativedelta(days=1)
         sponsorships_with_birthday_tomorrow = \
             self._get_sponsorships_with_child_birthday_on(tomorrow)
+
+        sponsorships_to_avoid = {}
+
+        for sponsorship in sponsorships_with_birthday_tomorrow:
+            sponsorship_correspondences = self.env['correspondence'].search([
+                ('sponsorship_id', '=', sponsorship.id),
+                ('direction', '=', "Supporter To Beneficiary"),
+                ('scanned_date', '>=', fields.Date.to_string(
+                    datetime.now() - relativedelta(months=-2)))
```

Comment:
Double negation will lead to an addition!
```suggestion
                    datetime.now() - relativedelta(months=2)))
```